daemon/config/config.go

```diff
@@ -171,7 +171,8 @@ type CommonConfig struct {
 	Experimental bool `json:"experimental"` // Experimental indicates whether experimental features should be exposed or not
 
 	// Exposed node Generic Resources
-	NodeGenericResources string `json:"node-generic-resources,omitempty"`
+	// e.g: ["orange=red", "orange=green", "orange=blue", "apple=3"]
+	NodeGenericResources []string `json:"node-generic-resources,omitempty"`
```

Comment:
Do we have a test to test loading this configuration from a `daemon.json` configuration file?

Reply:
Just added one :)